behandlingsprosess/src/main/java/no/nav/ung/sak/domene/behandling/steg/vedtak/FatteVedtakTjeneste.java

```diff
@@ -79,6 +90,17 @@ public BehandleStegResultat fattVedtak(BehandlingskontrollKontekst kontekst, Beh
         return BehandleStegResultat.utførtUtenAksjonspunkter();
     }
 
+    private static boolean harUtførtAksjonspunktOgGodkjentAlleVurderinger(Aksjonspunkt fatterVedtakAksjonspunkt, Collection<Totrinnsvurdering> totrinnaksjonspunktvurderinger) {
+        return fatterVedtakAksjonspunkt.getStatus().equals(AksjonspunktStatus.UTFØRT) && !totrinnaksjonspunktvurderinger.isEmpty() && totrinnaksjonspunktvurderinger.stream().allMatch(Totrinnsvurdering::isGodkjent);
+    }
+
+    private static List<AksjonspunktDefinisjon> finnIkkeGodkjenteVurderinger(Collection<Totrinnsvurdering> totrinnaksjonspunktvurderinger) {
+        return totrinnaksjonspunktvurderinger.stream()
+            .filter(a -> !TRUE.equals(a.isGodkjent()))
```

Comment:
Sjekker du likheten med .equals fordi isGodkjent kan returnere null eller er den alltid true/false?

Reply:
Feltet er ein primitiv boolean, så då antok eg det, men eg ser at den er definert som not null i databasen

Reply:
Du får vurdere om du vil endre på det. 
Ser det er brukt samme måte flere steder.